desktop/main.dev.js

```diff
@@ -14,6 +14,7 @@ import log from 'electron-log';
 import { ipcConsts } from '../app/vars';
 import MenuBuilder from './menu';
 import FileManager from './fileManager';
+import DiskStorageManager from './DiskstorageManager';
```

Comment:
file name is diskStorageManager so import will fail, did you tested it?

Reply:
mistake. fixed.